templates/cluster-template.yaml

```diff
@@ -202,3 +202,5 @@ spec:
             cloud-config: /etc/kubernetes/azure.json
             cloud-provider: azure
           name: '{{ ds.meta_data["local_hostname"] }}'
+      kubeadmConfigSpec:
+        useExperimentalRetryJoin: true
```

Comment:
you'll need to add this to flavors/base/cluster-template.yaml and then run make generate, this file is actually the output flavor generated by kustomize

Reply:
I have a changeset derived from `make generate` on the addition of this spec config here:

https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/templates/flavors/base/cluster-template.yaml#L116

Is that right?

Reply:
yes

Reply:
I take that back, you actually want to add it to https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/templates/flavors/default/machine-deployment.yaml#L47